erpnext/accounts/report/accounts_payable/accounts_payable.html

```
@@ -1 +1,220 @@
+                    {% if(filters.show_remarks) { %}
+                        <th style="text-align: left;">{%= __("Remarks") %}</th>
+                    {% } %}
+
+                    <th style="width: 10em; text-align: right;">{%= __("Age (Days)") %}</th>
+                    <th style="width: 10em; text-align: right;">{%= __("Invoiced Amount") %}</th>
+                    <th style="width: 11em; text-align: right;">{%= __("Outstanding Amount") %}</th>
+                </tr>
+            </thead>
+
+            <tbody>
+                {% for(var i=0, l=data.length; i<l; i++) { %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)